### PR TITLE
FIX: Swap object rename order

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

File: src/ansys/aedt/core/modeler/cad/object_3d.py [hunk 1]
```diff
@@ -1096,15 +1096,11 @@ def name(self):
...
     @name.setter
     def name(self, obj_name):
-        if obj_name not in self._primitives.object_names:
-            if obj_name != self._m_name:
-                vName = []
-                vName.append("NAME:Name")
-                vName.append("Value:=")
-                vName.append(obj_name)
+        if obj_name != self._m_name:
+            if obj_name not in self._primitives.object_names:
```
Comment (Alberto-DM) [inline src/ansys/aedt/core/modeler/cad/object_3d.py:1100]:
```suggestion
        if obj_name != self._m_name and obj_name not in self._primitives.object_names:
```
Why not joining the two if?

**Review by Alberto-DM — DISMISSED** [review]
Looks good to me, but it can be further simplified with a single `if` clause.

File: src/ansys/aedt/core/modeler/cad/object_3d.py (hunk 1 above)
Comment (SMoraisAnsys) [reply src/ansys/aedt/core/modeler/cad/object_3d.py:1100]: Agreeing with Alberto comment

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM, agreeing with Alberto though :)

**Review by Alberto-DM — APPROVED** [review]
LGTM

**Pull request merged**